recipes/pcapplusplus/all/conanfile.py

```diff
@@ -58,15 +65,34 @@ def _configure_sh_script(self):
         }[str(self.settings.os)]
 
     def _patch_sources(self):
-        if not self.options.get_safe("fPIC"):
-            tools.replace_in_file(os.path.join(self._source_subfolder, "PcapPlusPlus.mk.common"),
+        if not self.options.get_safe("fPIC") and self.settings.os != "Windows":
+            files.replace_in_file(self, os.path.join(self._source_subfolder, "PcapPlusPlus.mk.common"),
```

Comment:
This could be a patch since the change is static

Reply:
Existing logic, kind of outside of scope of this PR